CONTRIBUTING.md

```diff
@@ -121,6 +127,15 @@ the [Google Python Style guide](http://google.github.io/styleguide/pyguide.html)
 project code is examined using `pylint`, `flake8`, `mypy` and `black`, which will be run
 automatically in CI.
```

Comment:
sphinx build and error check is also done in CI. And we should recommend and teach developers to run the CI workflows locally to save time.